slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/PlainSQLTest.scala

```diff
@@ -12,90 +12,7 @@ class PlainSQLTest extends AsyncTest[JdbcTestDB] {
 
   case class User(id:Int, name:String)
 
-  //TODO convert to new API:
-  /*
```

Comment:
I have made the requisite changes to the tests and tried my best to convert old tests to ones using the new API. The two new tests are actually replacements of the existing test.
Is there something I have missed?